### PR TITLE
supervisor root

### PR DESCRIPTION
**Pull request by PainterHalver**
## Related Tickets
- [#65788](https://edu-redmine.sun-asterisk.vn/issues/65788)

## WHAT (optional)
- Thêm static page home cho supervisor.

## HOW
- Thêm controller, view supervisor/staticpages#home.
- Sửa lại redirect trong controller cũ.

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by PainterHalver** [comment]
ready

File: app/views/supervisor/static_pages/home.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <div class="container">
+      <h1 class="text-white"><%= t ".title" %></h1>
+    </div>
+  </div>
+
+  <div class="container mt-2">
+    <div class="row">
+      <div class="col-md-4">
+        <%= link_to t("layouts.header.manage_subjects"), supervisor_subjects_path,
+                    class: "width-full btn btn-info mt-2 py-2 f-lg"%>
```
Comment (ghost) [inline app/views/supervisor/static_pages/home.html.erb]:
```suggestion
                    class: "width-full btn btn-info mt-2 py-2 f-lg" %>
```

**Review by ghost — COMMENTED** [review]
cmt

**Comment by PainterHalver** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**